### PR TITLE
Some changes & bug fix

### PR DESCRIPTION
**Pull request by linkle69**
Made it so it exports the function directly and I updated the function name to be equip since it makes more sense in my opinion.

### CODE REVIEW

**Comment by linkle69** [comment]
I also fixed an issue where the server would reject the transaction if you tried to equip the totems too fast.
Now it will ignore the inventory check.

**Comment by LabGuy94** [comment]
The edits look great, although one thing needs to be changed to make the plugin function. Instead of defining the function inside of the `module.exports`, you need to define it before and use it in the `module.exports`:
What you did
```
module.exports = function inject(bot) {
    bot.autototem = {}
    bot.autototem.equip = () => {
        if (bot.inventory.slots[45] != null) return
        const totem = bot.inventory.findInventoryItem('totem_of_undying', null, null)
        if (totem) {
            bot.inventory.requiresConfirmation = false
            bot.equip(totem, 'off-hand')
        }
    }
}
```
Changes I made
```
function inject(bot) {
    bot.autototem = {}
    bot.autototem.equip = () => {
        if (bot.inventory.slots[45] != null) return
        const totem = bot.inventory.findInventoryItem('totem_of_undying', null, null)
        if (totem) {
            bot.inventory.requiresConfirmation = false
            bot.equip(totem, 'off-hand')
        }
    }
}
module.exports = {
    autototem: inject
}
```
All I did is just revert the change you did with the `module.exports`, everything else looks great. Please change your code to what I have suggested, thank you!

File: index.js [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
         const totem = bot.inventory.findInventoryItem('totem_of_undying', null, null)
-        if (totem) bot.equip(totem, 'off-hand')
+        if (totem) {
+            bot.inventory.requiresConfirmation = false
+            bot.equip(totem, 'off-hand')
+        }
     }
 }
-
-module.exports = {
```
Comment (LabGuy94) [inline index.js:10]: Please don't remove this and keep the `module.exports` the way it was, it does not work otherwise

**Review by LabGuy94 — CHANGES_REQUESTED** [review]
The edits look great, although one thing needs to be changed to make the plugin function. Instead of defining the function inside of the `module.exports`, you need to define it before and use it in the `module.exports`:
What you did
```
module.exports = function inject(bot) {
    bot.autototem = {}
    bot.autototem.equip = () => {
        if (bot.inventory.slots[45] != null) return
        const totem = bot.inventory.findInventoryItem('totem_of_undying', null, null)
        if (totem) {
            bot.inventory.requiresConfirmation = false
            bot.equip(totem, 'off-hand')
        }
    }
}
```
Changes I made
```
function inject(bot) {
    bot.autototem = {}
    bot.autototem.equip = () => {
        if (bot.inventory.slots[45] != null) return
        const totem = bot.inventory.findInventoryItem('totem_of_undying', null, null)
        if (totem) {
            bot.inventory.requiresConfirmation = false
            bot.equip(totem, 'off-hand')
        }
    }
}
module.exports = {
    autototem: inject
}
```
All I did is just revert the change you did with the `module.exports`, everything else looks great. Please change your code to what I have suggested, thank you!

File: index.js (hunk 1 above)
Comment (linkle69) [reply index.js:10]: ok

**Review by LabGuy94 — APPROVED** [review]
Great

**Pull request merged**